### PR TITLE
Complete basically everything except carousel

### PR DESCRIPTION
**Pull request by RyanL123**

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            display="flex"
+            flexDirection="column"
+            alignItems="center"
+        >
+            <Box px="10vw">
+                <Title
+                    sub="Lorem ipsum dolor"
+                    main="Lorem ipsum dolor sit amet"
+                />
+                <Text color="#BDBDBD" maxWidth="600px" mt="25px" mb="100px">
```
Comment (jhthenerd) [inline src/components/Footer.js:31]: maybe parameterize the color? idk

File: src/components/SEO.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            content="FraserHacks 2020 coming soon! Hosted at John Fraser Secondary School"
+        />
+        {/* Open Graph Protocol */}
+        <meta property="og:title" content="Fraser Hacks 2019 Recap" />
+        <meta
+            property="og:description"
+            content="FraserHacks 2020 coming soon! Hosted at John Fraser Secondary School"
+        />
+        <meta property="og:type" content="website" />
+        <meta property="og:image" content="/banner.png" />
```
Comment (jhthenerd) [inline src/components/SEO.js:18]: wait does pulling it from assets not work?

File: src/components/Title.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import { IsMobile } from "../util/mediaQuery"
+import { Heading, Box } from "@chakra-ui/core"
+
+export default ({ sub, main }) => (
+    <Box textAlign="center">
+        <Heading color="#298DC3" size={IsMobile() ? "md" : "lg"}>
```
Comment (jhthenerd) [inline src/components/Title.js:7]: parameterize color? also why u no use breakpoints they're literally built into chakra

File: src/components/Video.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    <Box
+        textAlign="center"
+        display="flex"
+        flexDirection="column"
+        alignItems="center"
+        px="10vw"
+        pb="100px"
+    >
+        <Title sub="Lorem ipsum dolor" main="Lorem ipsum dolor sit amet" />
+        <Text color="#BDBDBD" marginTop="25px" maxWidth="600px">
```
Comment (jhthenerd) [inline src/components/Video.js:16]: parameterize color

File: src/components/Video.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        <Heading
+            size="2xl"
+            color="white"
+            fontSize={IsMobile() ? "36px" : "72px"}
+            mt="50px"
+            mb="25px"
+        >
+            123,456,789
+        </Heading>
+        <Text color="#BDBDBD" maxWidth="600px">
```
Comment (jhthenerd) [inline src/components/Video.js:29]: also parameterize this

File: src/pages/index.js [hunk 6]
```diff
@@ -1,11 +1,19 @@
...
+import SEO from "../components/SEO"
 import Hero from "../components/Hero"
+import Carousel from "../components/Carousel"
+import Video from "../components/Video"
+import Footer from "../components/Footer"
 import { Box } from "@chakra-ui/core"
 
 export default () => {
     return (
         <Box background="#131D24">
```
Comment (jhthenerd) [inline src/pages/index.js:11]: params

File: src/util/mediaQuery.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
+import { useMediaQuery } from "react-responsive"
+
+export const IsMobile = () => {
+    const mobile = useMediaQuery({ query: "(max-width: 960px)" })
```
Comment (jhthenerd) [inline src/util/mediaQuery.js:4]: bruh just use breakpoints

**Review by jhthenerd — APPROVED** [review]
you could parameterize some shit but its not blocking and it seems to render fine. plus its just a one-pager that doesn't need to be maintained so I'll let it slide.

**Comment by jhthenerd** [comment]
@davidli3100 thoughts?

File: src/components/SEO.js (hunk 2 above)
Comment (RyanL123) [reply src/components/SEO.js:18]: apparently it has to be in static but I'm pretty sure it has to be an actual URL, we'll see what happens when it's on netlify

**Review by jhthenerd — CHANGES_REQUESTED** [review]
Nevermind, SEO is dead.
Please implement [Gatsby Plugin React Helmet](https://www.gatsbyjs.com/plugins/gatsby-plugin-react-helmet/) and [Gatsby Plugin Manifest](https://www.gatsbyjs.com/plugins/gatsby-plugin-manifest/)

**Comment by davidli3100** [comment]
Looks good but the photo at the bottom is still blurry asf @RyanL123 consider redoing it?

File: gatsby-config.js [hunk 8]
```diff
@@ -9,9 +9,40 @@ module.exports = {
...
+        },
+        {
+            resolve: `gatsby-plugin-chakra-ui`,
+            options: {
+                isResettingCSS: true,
+                isUsingColorMode: false,
+            },
+        },
+        {
+            resolve: `gatsby-plugin-manifest`,
```
Comment (jhthenerd) [inline gatsby-config.js:29]: Fix your favicon stuff, remove the custom implementation you have rn and replace with the built-in favicon generation functionality

See: https://www.gatsbyjs.com/plugins/gatsby-plugin-manifest/#automatic-mode-configuration

**Review by jhthenerd — CHANGES_REQUESTED** [review]
Just fix the favicon implementation and we should be good

**Review by jhthenerd — APPROVED** [review]
LGTM

File: src/components/Slider.js [hunk 9]
```diff
@@ -0,0 +1,171 @@
+import React from "react"
+import { Grid } from "@chakra-ui/core"
+import { graphql, useStaticQuery } from "gatsby"
+import Img from "gatsby-image"
+
+const Slider = () => {
```
Comment (err53) [inline src/components/Slider.js:6]: Size images properly, there's no need for a slider to be 1000 px across. Maybe like a few hundred px max.

**Pull request merged**